frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -36,8 +38,12 @@ interface IFeatureStrategyMenuProps {
 const StyledStrategyMenu = styled('div')(({ theme }) => ({
     flexShrink: 0,
     display: 'flex',
+    width: '100%',
     flexFlow: 'row',
     gap: theme.spacing(1),
+    '& > :nth-child(2)': {
```

Comment:
CSS is ~~insane~~ ~~mad~~ ~~stupid~~ elegant. You can just kick the 2nd element down the flex-box row